crawl-ref/source/files.cc

```diff
@@ -2085,6 +2091,53 @@ bool get_save_version(reader &file, int &major, int &minor)
     return true;
 }
 
+static bool _convert_obsolete_species()
+{
+    // At this point the character has been loaded but not resaved, but the grid, lua, stashes, etc have not been.
+#if TAG_MAJOR_VERSION == 34
+    if (you.species == SP_LAVA_ORC)
+    {
+        if (!yes_or_no("This <red>Lava Orc</red> save game cannot be loaded as-is. If you "
+                       "load it now, your character will be converted to a Hill Orc. Continue?"))
+        {
+            you.save->abort(); // don't even rewrite the header
+            delete you.save;
+            you.save = 0;
+            end(0, false, "Please load the save in an earlier version if you want to keep it as a Lava Orc.\n");
+        }
+        wizard_change_species_to(SP_HILL_ORC);
```

Comment:
Should we really be doing this with wizmode functions? It's probably okay I guess, but seems a little weird. Also, isn't it possible to compile without wizmode support? In those cases this would break, right? I think we could take this one function out of wiz-you and move it into someplace else, since we now need it even for non-wizard games.